### ISSUE TITLE
Low Poly Basic Geometry in RobotModel

### ISSUE THREAD

**Issue by dwffls** [ros2/rviz#897]
When loading URDF's with basic geometry like cylinders the shapes get very low in polygon count. And do not represent that what is shown in the tutorials on the wiki. Is this because of an RViz setting or something else?

Below are two images attached showing the difference. First is from ROS2 tutorials and second from my pc. 
Both have been genarated with the same command `ros2 launch urdf_tutorial display.launch.py model:='ros2 pkg prefix --share urdf_tutorial/urdf/01-myfirst.urdf`

![URDF Tutorial](https://raw.githubusercontent.com/ros/urdf_tutorial/ros2/images/myfirst.png)
![Screenshot from 2022-09-07 12-32-04](https://user-images.githubusercontent.com/1469494/188857213-5835b89e-2829-4c16-894e-63494202029e.png)

**Comment by clalancette** [comment]
Yeah, this looks like it was a problem going all the way back to Dashing.  In short, I'm not sure what is causing this, it needs to be investigated.
